tools/fix_auth.rb

```diff
@@ -11,14 +11,17 @@
   $LOAD_PATH.push(File.expand_path(File.join(__dir__, %w(.. gems pending))))
 end
 
-require File.expand_path('../../config/environment', __FILE__)
-
 require 'active_support/all'
 require 'active_support/concern'
 require 'fix_auth/auth_model'
 require 'fix_auth/auth_config_model'
 require 'fix_auth/models'
 require 'fix_auth/cli'
 require 'fix_auth/fix_auth'
+require 'more_core_extensions/all'
+require 'util/miq-logger'
+require 'vmdb/loggers'
+require 'vmdb/logging'
+require 'vmdb/settings'
```

Comment:
does it make sense to raise `if defined?(Rails)` ?  It feels like something ELSE in this chain can load environment and then FixAuth is broken again.